README.md

```diff
@@ -1,16 +1,27 @@
+    3.	Different operation executions can also change the time. Depending how the program was written/organized could increase or decrease the runtime. An obvious example would be using a nested loop when it could be done in a much more efficient way.
 
-- Suppose finding a particular element in a binary search tree with 1,000
+
+**- Suppose finding a particular element in a binary search tree with 1,000
   elements takes 5 seconds. Given what you know about the asymptotic complexity
   of search in a binary search tree, how long would you guess finding the same
-  element in a search tree with 10,000 elements takes? Explain your reasoning.
+  element in a search tree with 10,000 elements takes? Explain your reasoning.**
+    We know that, in general, a BST has a runtime of O(log<sub>2</sub>n) (in the most common case) which represents the height of the tree. When searching through 1000 elements, of course, no two runtimes will be the exact same. That said, we can generalize and get a rough idea of what it should be as we do in asymptotic analysis. We know that with O(logn), as input size increases exponentially, the time will increase linearly. This would mean that  1000x=10000 or x = log1000(10000) where x = 1.33333. So, if it took 5 seconds to search 1000 elements, we can guess that it will take a little under 7 seconds to compute 10000 elements. T(n) <= c log2n or 5sec = c log21000 where c can be found, as well as x = c log<sub>2</sub>10000 should result in 6 2/3 seconds, roughly. Linearly of course would be 10x as long (50 seconds).
+
 
-- You measure the time with 10,000 elements and it takes 100 seconds! List 3
+**- You measure the time with 10,000 elements and it takes 100 seconds! List 3
   reasons why this could be the case, given that reasoning with the asymptotic
-  complexity suggests a different time.
+  complexity suggests a different time.**
+    1.	There is some unknown operation it does that takes much longer to execute than the typical BST. This could be something like checking every node rather than utilizing the BST structure (with some, much slower type of search) or doing an additional operation like deleting/adding nodes or performing an operation as you search. These are the constant values we ignore when finding the runtime of an algorithm.
+    2.	Another reason (and most likely) could be that the binary search tree is skewed to one side or another, for example, the right side of the tree has many more elements than the left side(all nodes are larger than start node). This would increase the height and path it needs to take to find an element and thus, increase the complexity and time of the algorithm (O(n) depending on the desired node).
```

Comment:
Would that account for such a large difference?

Reply:
Machine or hardware problem could help with the difference, specifically pulling data from a file on the hard drive or running on a different machine. Background operations on the computer can affect this as well.

Reply:
I'm specifically referring to your second reason.